src/assets/data/featureEvent.json

```diff
@@ -1,21 +1,21 @@
 {
-    "FeatureEventName":"IWD'19 Jalandhar",
-    "EventImage":"IWD.jpg",
+    "FeatureEventName":"Flutter workshop",
+    "EventImage":"flutter.png",
     "EventDate":{
-        "Date":"31",
-        "Month":"3",
+        "Date":"23",
+        "Month":"11",
         "Year":"2019"
     },
     "EventTime":{
-        "StartTime":"09:00 AM",
-        "EndTime":"04:30 PM"
+        "StartTime":"12:30 PM",
+        "EndTime":"03:30 PM"
     },
-    "EventVenue":"Apeejay Institute of Management Technical Campus, Rama Mandi, Hoshiarpur Road, Jalandhar",
-    "EventVenueMapLink":"https://goo.gl/maps/4Z8pminYsp42",
-    "EventWebsite":"https://iwd.gdgjalandhar.com/",
-    "EventDescription":"International Women's Day (IWD) is marked on March 8th every year. In different regions, the focus of celebrations ranges from general respect and appreciation for women to a celebration of women's economic, political, and social achievements.",
-    "MeetupLink":"https://www.meetup.com/GDG-Jalandhar/events/253165595/",
-    "RegistrationLink":"https://docs.google.com/forms/d/e/1FAIpQLSciM-nL87uQd_it2ulou1mFv55aXqE9qSSnwL3No3fWhX8OzA/closedform",
-    "FBEventPageURL":"s"
+    "EventVenue":"Shift4 Payments Lithuania, A. Goštauto g. 40a, Vilnius",
+    "EventVenueMapLink":"https://goo.gl/maps/ViR6mZCdkmW3YZAy8",
+    "EventWebsite":"",
+    "EventDescription":"Flutter was released just 2 years ago, but it already gained huge following. But a lot of developers still have not heard or seen how Flutter works. \"Why the heck is everyone is so worked up about this Flutter thing?\" presentation will introduce this new technology in a simple way and answers questions like: \"What the heck is Flutter?\"  or \"Why the heck would anyone use it?\"",
+    "MeetupLink":"https://www.meetup.com/GDG-Vilnius/events/266283790/",
+    "RegistrationLink":"",
+    "FBEventPageURL":""
```

Comment:
Is this updated automatically?

Reply:
I don't think it is. I believe we can change it to show the earliest next event.